metadata-ingestion/tests/integration/sigma/golden_test_platform_instance_ingest.json

```diff
@@ -637,10 +806,6 @@
                     "id": "urn:li:dataPlatformInstance:(urn:li:dataPlatform:sigma,cloud_instance)",
                     "urn": "urn:li:dataPlatformInstance:(urn:li:dataPlatform:sigma,cloud_instance)"
                 },
-                {
-                    "id": "urn:li:container:abbebb5181bf9ba2d905d2dea7d8704d",
-                    "urn": "urn:li:container:abbebb5181bf9ba2d905d2dea7d8704d"
-                },
                 {
                     "id": "urn:li:container:084a2e283eddfc576ce70989b395a7d8",
                     "urn": "urn:li:container:084a2e283eddfc576ce70989b395a7d8"
```

Comment:
This looks suspicious. Workspace container should still be present in browsepath, whereas workbook container(last one) should be removed.